operations/cancellationpolicies.md

```diff
@@ -84,7 +84,7 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | Property | Type | Contract | Description |
```

Comment:
FYI: changes in this file are due to previous task (migration of cancellation policy page) when I did some changes in backend repo and didnt regenerate documentation 😅